common/milestone.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2014 The go-ethereum Authors
```

Comment:
Can we remove this as this is our file? 😅

Reply:
Yes,Actually It was being used in another implementation. I forgot to remove it.